Vagrantfile

```diff
@@ -55,6 +55,7 @@ Vagrant.configure("2") do |config|
   config.vm.provision "shell", path: "provisioning/install-code_aster.sh", privileged: false
   config.vm.provision "shell", path: "provisioning/install-dune.sh", privileged: false
   config.vm.provision "shell", path: "provisioning/install-paraview.sh", privileged: false
+  config.vm.provision "shell", path: "provisioning/install-julia.sh", privileged: false
```

Comment:
This file seems to be missing from the PR.

Reply:
Oh right, sorry, without this its a bit pointless. The commit message is also somehow broken, but it should be up to date now.